FIX: QA/QC of denoising not implemented (#460)
TheAxonLab/hcph-sops: docs/data-management/qaqc-general.md

# General QA/QC protocol

Following our protocols<sup>[1]</sup> and best practices<sup>[2]</sup>, we establish quality checkpoints after the outstanding stages of our neuroimaging pipeline.

> That is, we establish a checkpoint [after the conversion to BIDS](post-session.md#formal-qc), on the [unprocessed (before preprocessing) data](mriqc.md#visualizing-mriqcs-individual-reports), on the [preprocessed data](../processing/preprocessing.md#visualizing-fmripreps-individual-reports), and further downstream analysis points such as the [extracted FC](../processing/qaqc-criteria-FC.md).

Reviewer: ```suggestion
That is, we establish checkpoints [after the conversion to BIDS](post-session.md#formal-qc), on the [unprocessed (before preprocessing) data](mriqc.md#visualizing-mriqcs-individual-reports), on the [preprocessed data](../processing/preprocessing.md#visualizing-fmripreps-individual-reports), and further downstream, for instance, following [FC extraction](../processing/qaqc-criteria-FC.md).
```
